ops/migrations/000999_dids.down.sql

```diff
@@ -0,0 +1,102 @@
+-- Organizations
```

Comment:
@theosirian as part of your rebase, can you set the migration to n+1 based on what is under ops/migrations?